HopsanGUI/LogDataHandler2.cpp

```diff
@@ -996,6 +995,101 @@ void LogDataHandler2::importFromPlainColumnCsv(QString importFilePath, const QCh
     limitPlotGenerations();
 }
 
+void LogDataHandler2::importFromPlainRowCsv(QString importFilePath, const QChar separator, const int linesToSkip, const int timeRow)
+{
+    if(importFilePath.isEmpty())
+    {
+        importFilePath = QFileDialog::getOpenFileName(0,tr("Choose .csv File"),
+                                                       gpConfig->getStringSetting(CFG_PLOTDATADIR),
+                                                       tr("Comma-separated values files (*.csv)"));
+    }
+    if(importFilePath.isEmpty())
+    {
+        return;
+    }
+
+    QFile file(importFilePath);
+    QFileInfo fileInfo(file);
+    gpConfig->setStringSetting(CFG_PLOTDATADIR, fileInfo.absolutePath());
+
+    bool parseOk = true;
+
+    if (file.open(QFile::ReadOnly))
+    {
+        QStringList allNames, allAlias, allUnits;
+        QList< QVector<double> > allDatas;
+        int nDataValueElements = -1;
+        QTextStream contentStream(&file);
+        while (!contentStream.atEnd())
+        {
+            //! @todo this is not the most clever and speedy implementation
+            QStringList lineFields = contentStream.readLine().split(separator);
+            if (lineFields.size() > linesToSkip)
```

Comment:
This looks odd, linesFields is columns I belive, and it is compared with a variable linesToSkip, Should it be columnsToSkip?

Reply:
I renamed linesToSkip to rowsToSkip/columnsToSkip in both improt functions to make it more clear.